src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -50,9 +50,19 @@ def load_arguments(self, _):
     isolated_sku_arg_type = CLIArgumentType(help='The Isolated pricing tiers, e.g., I1 (Isolated Small), I2 (Isolated Medium), I3 (Isolated Large)',
                                             arg_type=get_enum_type(['I1', 'I2', 'I3']))
 
+    functionapp_runtime_to_version = {}
+    for version in RUNTIME_TO_IMAGE_FUNCTIONAPP.values():
+        for runtime, val in version.items():
+            if runtime in functionapp_runtime_to_version:
+                functionapp_runtime_to_version[runtime] = set(functionapp_runtime_to_version[runtime]).union(set(val.keys()))
+            else:
+                functionapp_runtime_to_version[runtime] = set(val.keys())
```

Comment:
This can be simplified as
```python
functionapp_runtime_to_version[runtime] = functionapp_runtime_to_version.get('runtime', set()).union(val.keys())
```